tool/cli.js

```diff
@@ -650,21 +650,13 @@ program
       default: MAX_GOOGLE_ANALYTICS_URIS,
     }
   )
-  .argument("csvfile", "Google Analytics pageviews CSV file", {
-    validator: (value) => {
-      if (!fs.existsSync(value)) {
-        throw new Error(`${value} does not exist`);
-      }
-      return value;
-    },
-  })
   .action(
-    tryOrExit(async ({ args, options, logger }) => {
+    tryOrExit(async ({ options, logger }) => {
```

Comment:
This command still speaks about "Google Analytics".

Reply:
👍